explore/src/main/scala/explore/tabs/TargetTabContents.scala

```diff
@@ -765,15 +764,6 @@ object TargetTabContents extends TwoPanels:
           React.Fragment(
             // TODO switch to prime react
             Toast(Toast.Position.BottomRight).withRef(toastRef.ref),
-            if (active.get.value) {
-              <.div(
-                ^.cls := "ui active dimmer",
-                Loader(
-                  active = true,
-                  "Cloning observation..."
-                )
-              )
-            } else EmptyVdom,
```

Comment:
I was never able to actually observe this `Loader`. I think maybe it got overridden by the `asterismGroupsWithObs.render`? The `+Obs` button is disabled and `loading`, anyway.